importers/yelp.ca/download_html.py

```diff
@@ -0,0 +1,124 @@
+import os
+import lxml.html as html
+import urllib2
+import re
+
+
+yelp_base_url = 'https://www.yelp.ca/'
```

Comment:
There is an import tool for yelp.com already.  Why are you making a separate tool for yelp.ca?  Is the postal code or zipcode named differently for yelp.ca than yelp.com?  Is the site programmed differently?